cypress/e2e/awx/administration/notifiers/notifiersTabs.cy.ts

```diff
@@ -242,11 +242,11 @@ function testToggle(
 
     // reload page to check if the toggle is working and try to disable it
     cy.reload();
-    cy.get(`[aria-label="${type_disable}"]`, { timeout: 15000 }).click();
+    cy.get(`[aria-label="${type_disable}"]`, { timeout: 25000 }).click();
```

Comment:
I think we can do this without using reload
see if this code is useful which I have written for Management Job Schedules and Notification tabs
```
cy.get('.pf-v5-c-switch__label.pf-m-on')
                  .should('have.text', 'Schedule enabled')
                  .should('be.visible');
```

                // Toggle the schedule to disabled
                cy.getByDataCy('toggle-switch').click();

                // Assert the schedule is disabled
                cy.get('.pf-v5-c-switch__label.pf-m-off')
                  .should('have.text', 'Schedule disabled')
                  .should('be.visible');`

Reply:
or this
`//enable and disable start, success and failure switches of the notifier
              cy.toggleAndAssert(['start', 'success', 'failure']);`

Reply:
Im not sure if this will not lead to problems, because we will immidiately call switch on and switch off and that could be processed in parallel in backend and thus leading to problems. But ok, lets try it without reload.